apps/mobile/src/libs/components/gradient-slider/gradient-slider.tsx

```diff
@@ -0,0 +1,139 @@
+import Slider from "@react-native-community/slider";
+import React from "react";
+
+import {
+	Icon,
+	LinearGradient,
+	SliderSection,
+	Text,
+	View,
+} from "~/libs/components/components";
+import { BaseColor } from "~/libs/enums/enums";
+import {
+	generateSliderGradientColors,
+	generateSliderGradientLocations,
+} from "~/libs/helpers/helpers";
+import { useCallback, useState } from "~/libs/hooks/hooks";
+import {
+	type colorToGradientColors,
+	directionToGradient,
+} from "~/libs/maps/maps";
+import { globalStyles } from "~/libs/styles/styles";
+import { type LayoutChangeEvent } from "~/libs/types/types";
+
+import {
+	SLIDER_INITIAL_WIDTH,
+	SLIDER_MAX_PERCENT,
+	SLIDER_STEP_VALUE,
+} from "./libs/constants/slider-config";
+import {
+	SLIDER_ICON_TRANSLATE_OFFSET,
+	SLIDER_MARKER_ICON_SIZE,
+	SLIDER_SECTION_TRANSLATE_OFFSET,
```

Comment:
Combine to enum

Reply:
I added the enum locally in the component folder because it is only for the slider